snuba/datasets/plans/single_storage.py

```diff
@@ -7,19 +7,27 @@
     StorageQueryPlanBuilder,
 )
 from snuba.datasets.storage import QueryStorageSelector, ReadableStorage
+from snuba.datasets.plans.translators import CopyTranslator
 
 # TODO: Importing snuba.web here is just wrong. What's need to be done to avoid this
 # dependency is a refactoring of the methods that return RawQueryResult to make them
 # depend on Result + some debug data structure instead. Also It requires removing
 # extra data from the result of the query.
 from snuba.web import RawQueryResult
-from snuba.query.query_processor import QueryProcessor
+from snuba.query.physical import Query
+from snuba.query.processors.physical import QueryProcessor
 from snuba.request import Request
+from snuba.request.request_settings import RequestSettings
```

Comment:
Is sorting here due to circular imports?

Reply:
No. It is due to codemod. After codemodding everything to logical and physical I did not notice the order was not right anymore.

Reply:
fixed